discover/portal/articles/04-business-productivity/00-intro.markdown

```diff
@@ -1,3 +1,68 @@
-# Forms and Workflow [](id=business-productivity)
+# Forms and Lists
 
-This is a placeholder. The real intro will be written soon. 
+When you need a form in @product@, what you really need is data. @product@ has
+two applications for building forms to collect precisely the data you need:
+
+1.  [Liferay Forms](/discover/portal/-/knowledge_base/7-0/collecting-information-from-users): @product@'s primary form building application lets you
+    build forms ranging from the simplest one or two question survey to the most
+    complex, multi-page, homeowners insurance application form featuring form
+    rules and lists populated by a REST data provider.
+
+2.  [Dynamic Data
+    Lists(DDL)](/discover/portal/-/knowledge_base/7-0/creating-simple-applications):
+    Provides a user interface tool for building reusable form- and list-based
+    applications intended for display on pages in @product@. It's the ability to
+    mold the display using [templates](/discover/portal/-/knowledge_base/7-0/using-templates-to-display-forms-and-lists) that sets DDL apart.
+
++$$$
+
+**Kaleo Forms:** If you're a Liferay Digital Enterprise customer, there's a
+third form building tool in @product@ called [Kaleo
+Forms](/discover/portal/-/knowledge_base/7-0/workflow-forms). It integrates form
+building with workflow to create form based business processes, like a
+Conference Room Checkout Form, or a Support Ticket Process so support tickets go
+through the proper channels on their way to resolution. Read more about Kaleo
+Forms in the [workflow
+section](/discover/portal/-/knowledge_base/7-0/using-workflow).
+
+$$$
+
+## Which Form Builder Should I Use?
+
+Liferay Forms gives you a lot of control over how your form is displayed,
+allowing you to make [form
+pages](/discover/portal/-/knowledge_base/7-0/creating-advanced-forms#creating-form-pages),
+and adjust the [form
+layout](/discover/portal/-/knowledge_base/7-0/creating-advanced-forms#form-layouts)
+of the fields. These styling elements are enough for most forms. Over time, more
+features will be added to Liferay Forms so that DDL might someday become
+obsolete, in favor of a single form building application, Liferay Forms. If
+you're a heavy user of DDL, don't worry, Liferay is committed to preserving and
+enhancing the features provided by DDL; they just might be moved to Liferay Forms. Now is the time to familiarize yourself with Liferay Forms and
```

Comment:
I've become a little bit uncomfortable with this sentence, once, from what I know, we won't improve DDL, actually, the idea is deprecate it. I'm just worried about create an incorrect expectations from the customers side, you know?

Reply:
What I meant by this part is that if customers are heavy DDL users, the features will be preserved even when DDL goes away for good. That's true, isn't it? I'll try to reword it to make this more clear. @natocesarrego

Reply:
I think it's true, but as this is something that will happen in the future, I'm not sure if ALL features from DDL will be available. Maybe we can double check it with Charlles or the PT.

What do you think?